spec/javascripts/components/projects/project_board_spec.js

```diff
@@ -284,4 +286,49 @@ describe('<ProjectBoard />', () => {
       expect(wrapper.find('[data-id="project-loading"]')).toExist();
     });
   });
+
+  describe('when have epicStories', () => {
+    it('renders epic column', () => {
+      const props = {
+        epicStories: [storyFactory()],
+        projectBoard: { 
+          isFetched: true,
+          search: {
+            loading: false
+          },
+          reverse: false,
+          visibleColumns: {
+            backlog: true,
+            done: true,
+            chillyBin: true
+          }
+        },
+      };
+      const wrapper = render(props);
+
+      expect(wrapper.find('[data-id="epic-column"]')).toExist();
+    });
+  });
+
+  describe('when have not epicStories', () => {
```

Comment:
What do you think about changing to this and passing an empty array instead of not passing episStories at all?

```suggestion
  describe('when epicStories is empty', () => {
```

Reply:
Nice! Thank you.